x/mint/keeper/inflation.go

```diff
@@ -0,0 +1,30 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k Keeper) HandleInflationChange(ctx sdk.Context) (inflationRate sdk.Dec, err error) {
```

Comment:
From the function, i understand we support increasing inflation functions as well. 
I thought, from the PRD, inflation was always a decreasing function, but just asking to be sure.

Reply:
it depends on the target inflation, relatives to current inflation